qt/MainWindow.h

```diff
@@ -100,7 +100,7 @@ public slots:
     QMenu* createStatsModeMenu();
     void initStatusBar();
 
-    void clearSelection();
+    void clearSelection() const;
```

Comment:
Despite adding `const` is possible, the method is not logically constant and I cannot imagine the need to call it in constant context...

Reply:
Reverted in 6daf368dd76965d6d495d62444aaa6db48737d71